src/SessionManager.ts

```diff
@@ -117,19 +145,39 @@ export class SessionManager extends EventEmitter {
     return session;
   }
 
+  /**
+   * @param sessionId A unique session identifier.
+   * @returns true if a session associated to the given ID exists.
```

Comment:
```suggestion
   * @returns A Promise resolving to true if a session associated to the given ID exists, and false if not.
```

To emphasise that a non-existent Session won't result in the Promise rejecting.